src/js/share.js

```diff
@@ -392,10 +397,9 @@
 				.then(function (data) {
 					if (data.success) {
 						const code = data.data.shareCode;
-
 						const join = (window.location.href.indexOf("?") > -1) ? "&" : "?";
 
-						window.history.pushState({}, undefined, window.location.href + join + "share_code=" + code);
+						window.history.pushState({}, undefined, window.location.href.split('#')[0] + join + "share_code=" + code);
```

Comment:
Could we use `window.location.href + '?' + query-string.stringify + '#' + window.location.hash` or something?